Adds additional projects verified using Gobra (#741)
viperproject/gobra: README.md

## Projects verified with Gobra
- [VerifiedSCION](https://github.com/viperproject/VerifiedSCION)
- [Security of protocol implementations via refinement w.r.t. a Tamarin model](https://github.com/viperproject/protocol-verification-refinement). In particular, implementations of the signed Diffie-Hellman and WireGuard protocols have been verified.

> - [Security of protocol implementations relying only on Gobra](https://github.com/viperproject/SecurityProtocolImplementations). In particular, implementations of the Needham-Schroeder-Lowe, signed Diffie-Hellman, and WireGuard protocols have been verified.

Reviewer: ```suggestion
- [Security of protocol implementations verified entirely within Gobra](https://github.com/viperproject/SecurityProtocolImplementations). In particular, implementations of the Needham-Schroeder-Lowe, signed Diffie-Hellman, and WireGuard protocols have been verified.
```

As a suggestion. I am not sure what statement you are trying to make. Based on the previous point, I assume you want to emphasize that you showed correctness of the code and the protocol induced by the code using Gobra only, in contrast to using tools like Tamarin. For me, "relying" does not convey this information.

Author: Thanks, that's better
